### PR TITLE
Chart

### PR DESCRIPTION
**Pull request by zhengmar3**
Added various chart samples using react-chartjs-2 and ant-design/chart library.

image
image
image

### CODE REVIEW

**Review by yarivus — COMMENTED** [review]
Reviewed code partially.
Thanks for the screenshots.
LGTM

File: mui-template-app/src/components/ConfigurationCard/ConfigurationCard.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              variant="caption"
+              component="div"
+              color="text.secondary"
+              gutterBottom
+            >
+              Code review requests from your team members
+            </Typography>
+          </div>
+          <FormControlLabel
+            label=""
```
Comment (Evgeny-Timoshin-R3) [inline mui-template-app/src/components/ConfigurationCard/ConfigurationCard.tsx:92]: Is this missing label content?

File: mui-template-app/src/components/StockChart/StockChart.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { Typography } from "@mui/material";
+
+export function StockChart() {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    asyncFetch();
+  }, []);
+
+  const asyncFetch = () => {
```
Comment (Evgeny-Timoshin-R3) [inline mui-template-app/src/components/StockChart/StockChart.tsx:12]: Same as one of the previous comments about using async await and placing inside useEffect.

**Review by Evgeny-Timoshin-R3 — COMMENTED** [review]
Minor stuff. Looks good. Charts are cool.

File: mui-template-app/src/components/ConfigurationCard/ConfigurationCard.tsx (hunk 1 above)
Comment (zhengmar3) [reply mui-template-app/src/components/ConfigurationCard/ConfigurationCard.tsx:92]: I have to left it empty. I don't need a label as user can understand the action with the text on the left and the toggle color, but mui required this prop to be there.

File: mui-template-app/src/components/AnimatedLineChart/AnimatedLineChart.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import { Line, LineConfig } from "@ant-design/plots";
+
+export function AnimatedLineChart() {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    asyncFetch();
+  }, []);
+
+  const asyncFetch = () => {
```
[the remark this answers is not in the document]
Comment (zhengmar3) [reply mui-template-app/src/components/AnimatedLineChart/AnimatedLineChart.tsx]: Ah agree! overlooked it as it came from the sample of the library directly. Changed all instances to best practice now.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by Evgeny-Timoshin-R3 — APPROVED** [review]
LGTM

**Pull request merged**